cluster/gce/gci/configure-helper.sh

```diff
@@ -753,6 +752,9 @@ function start-kube-apiserver {
   params+=" --tls-cert-file=/etc/srv/kubernetes/server.cert"
   params+=" --tls-private-key-file=/etc/srv/kubernetes/server.key"
   params+=" --token-auth-file=/etc/srv/kubernetes/known_tokens.csv"
+  if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
```

Comment:
Right. As it works today, the file persists across upgrades. This would allow an upgrade w/ unset env vars make the apiserver stop using the already existing file.